web-console/knikubevirt/Create-vm/create-vm.md

```diff
@@ -0,0 +1,163 @@
+# Create A Virtual Machine
+
+## Launching the Wizard
+
+![Create VM](img/Create-VM.png)
```

Comment:
👍 

> Looks good @matthewcarleton! Just left a minor note about the empty state. Also I'm not sure if that info icon is correct since usually it's not filled in. I don't think you need to change this in the mocks as long as we implement with the correct icon though. We can check with @mceledonia.

yup just talked to him about this and we are moving back to the old icons. I'm updating now :)

Reply:
@beanh66 I've updated the empty state to show a secondary action in there.
Also, I've moved the "create vm" dropdown to the right, which is going to work better visually, when we add the actions dropdown it could just sit next to it. WDYT?